src/UNL/VisitorChat/Controller.php

```diff
@@ -53,6 +53,8 @@ class Controller extends \Epoch\Controller
     
     public static $pagetitle = "UNLchat";
     
+    public static $badWords = array('fuck', 'shit', 'pussy', 'ejaculate');
```

Comment:
Wow, I would _much_ prefer this to be in a private data file, rather than in the public source.